packages/frontend/src/lib/libp2p.ts

```diff
@@ -45,8 +51,8 @@ export async function startLibp2p() {
     }),],
     connectionEncryption: [noise()],
     connectionManager: {
-      maxConnections: 200,
-      minConnections: 1,
+      maxConnections: 10,
+      minConnections: 50,
     },
```

Comment:
`minConnections` needs to be less than `maxConnections`.

Is there a reason you're overriding this?  If not I'd just remove the whole `connectionManager` block.

Reply:
removed, I'm not sure why it was added. @maschad any idea?